api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -176,3 +185,45 @@ message DeleteAllAccessListMembersRequest {
   reserved 1;
   reserved "access_list";
 }
+
+// ListAccessListReviewsRequest is the request for getting paginated access list reviews for a particular access list.
+message ListAccessListReviewsRequest {
+  // access_list is the name of the access list that we're listing reviews for.
+  string access_list = 1;
+
+  // page_size is the size of the page to request.
+  int32 page_size = 2;
+
+  // next_token is the page token.
+  string next_token = 3;
+}
+
+// ListAccessListReviewsResponse is the response for getting paginated access list rviews for a particular access list.
+message ListAccessListReviewsResponse {
+  // reviews is the list of access list reviews.
+  repeated Review reviews = 1;
+
+  // next_token is the next page token.
+  string next_token = 2;
+}
+
+// CreateAccessListReviewRequest is the request for creating an access list review.
+message CreateAccessListReviewRequest {
+  // review is the actual review to create.
+  Review review = 1;
+}
+
+// CreateAccessListReviewResponse is the response for creating an access list review.
+message CreateAccessListReviewResponse {
+  // review_name is the name of the review that was just created.
+  string review_name = 1;
```

Comment:
What's review "name"? Is it basically ID?

Reply:
Yes, exactly.